app/src/main/AndroidManifest.xml

```diff
@@ -9,15 +9,26 @@
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
 
-    <application android:allowBackup="true" android:icon="@mipmap/ic_launcher"
-        android:label="@string/app_name" android:supportsRtl="true" android:theme="@style/AppTheme">
-        <activity android:name=".MainActivity"
+    <application
+        android:allowBackup="true"
```

Comment:
This will not work since our software does not allow proprietary software (e.g. Google accounts) - https://developer.android.com/guide/topics/data/autobackup.html